packages/material-ui-utils/src/capitalize.ts

```diff
@@ -0,0 +1,13 @@
+// TODO: error TS7016: Could not find a declaration file for module '../macros/MuiError.macro'. '/tmp/material-ui/packages/material-ui-utils/macros/MuiError.macro.js' implicitly has an 'any' type.
+// import MuiError from '../macros/MuiError.macro';
```

Comment:
Could you `@ts-expect-error` this instead for now so that we don't sacrifice the runtime behavior for typings?

Reply:
After I added the import I've I started seeing this circular problem:

```
./src/index.js → build/umd/material-ui.development.js...
[!] Error: Circular dependency: ../material-ui-utils/src/index.ts -> ../material-ui-utils/src/capitalize.ts -> ../material-ui-utils/src/index.ts
Error: Circular dependency: ../material-ui-utils/src/index.ts -> ../material-ui-utils/src/capitalize.ts -> ../material-ui-utils/src/index.ts
    at Object.onwarn (/tmp/material-ui/packages/material-ui/scripts/rollup.config.js:74:11)
    at Object.config.onwarn.warning [as onwarn] (/tmp/material-ui/node_modules/rollup/dist/shared/mergeOptions.js:98:25)
    at Object.config.onwarn.warning [as onwarn] (/tmp/material-ui/node_modules/rollup/dist/shared/rollup.js:19484:20)
    at Graph.sortModules (/tmp/material-ui/node_modules/rollup/dist/shared/rollup.js:18913:26)
    at Graph.build (/tmp/material-ui/node_modules/rollup/dist/shared/rollup.js:18826:14)
```

Reply:
So the macro runtime is actually broken. I think it always assumes that it is imported from another library. I'll take a look. This is blocking a clean merge for now.

Reply:
Fixed in https://github.com/mnajdova/material-ui/pull/11

Reply:
Merged